doc/getting-started.md

```diff
@@ -207,6 +213,11 @@ file which provides environment variables for the Docker containers:
 echo "KCI_API_TOKEN=<your token>" >> .env
 ```
 
+> **Note** Admin API token created in the previous section can be used for
+> initial Pipeline instance testing. In production environment the best
+> practice would be to create a separate (non-admin) user with required scopes
+> and generate a new API token for that user.
+
```

Comment:
I don't think it would be a good idea to use API token with admin scope in the pipeline. The `admin` token should be used only in the API to perform admin-related activities like user and group management.

Reply:
Sure, but for a local developer setup with a single user it doesn't really matter.  It's just a bit better to have a plain user token to exercise the permission checks I guess.  It's a bit like Docker images with root as the default user, not great but with no consequence outside the container anyway.

Reply:
Yes, it won't be an issue for the local setup.